### ISSUE TITLE
Add hover text for household size indicator

### ISSUE THREAD

**Issue by piratefsh** [crownheightsaid/mutual-aid-app#116]
![Screen Shot 2020-07-08 at 1 32 53 PM](https://user-images.githubusercontent.com/1868400/86951490-9cfa0780-c11f-11ea-910e-cfa8a0ec3690.png)

The pink chip indicates household size but doesn't have any alt text right now. For usability, we should add hover text that says something like `Household size of 2`.

Location of source: https://github.com/crownheightsaid/mutual-aid-app/blob/master/src/webapp/components/RequestPopup.js#L83
